README.md

```diff
@@ -297,6 +297,28 @@ This should produce an installer exe, pkg or rpm+deb file in the `dist/` directo
 
 Building this requires at least 1GB of memory.
 
+#### Apple ARM64 (aka Apple Silicon)
```

Comment:
The `prost` version we depend on doesn't come with the arm64 `protoc` binary bundled and unfortunately we can't update it yet since it's incompatible with some of our other dependencies. A simple workaround is to just install protobuf `brew install protobuf`. It would be nice if this workaround is added to the readme until we can update `prost`.

Reply:
Indeed, I forgot that I had to install `protobuf`. Thanks for reminding me.

Reply:
Done